bedrock/base/templates/macros.html

```diff
@@ -150,15 +150,14 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
   {% set fxa_queries = [
     'form_type=' ~ form_type,
     'entrypoint=' ~ entrypoint,
-    'context=fx_desktop_v3',
```

Comment:
Ok, I double checked this flow and I can't see anywhere in the bug that suggests removing this parameter?

It seems to be required in order to trigger the correct flow, at least as far as i can test (please test below). Only when including `context` did I see this page below (notice the `Don't sync this device` link):

![image](https://user-images.githubusercontent.com/400117/69710763-e00aeb80-10f7-11ea-8fb2-b67cdb4355ec.png)

This matches up with the slides linked in https://bugzilla.mozilla.org/show_bug.cgi?id=1595829#c5, which is something I missed in my review earlier (sorry).

You can test the two flows here:

- No `service` and no `context` param (FAIL): https://accounts.firefox.com/signup?form_type=button&entrypoint=mozilla.org-whatsnew71&utm_source=mozilla.org-whatsnew71&utm_medium=referral&utm_campaign=whatsnew71&utm_content=top-cta&deviceId=3dc1464bdb734a19a7cf0c8569eb2c95&flowBeginTime=1574847318905&flowId=3fefe938bb86ff4c315e44aef31bbbf408adeb3f1b9111bbb2bc2cab7824d1de
- Just no `service` param (PASS): https://accounts.firefox.com/signup?form_type=button&entrypoint=mozilla.org-whatsnew71&context=fx_desktop_v3&utm_source=mozilla.org-whatsnew71&utm_medium=referral&utm_campaign=whatsnew71&utm_content=top-cta&deviceId=3dc1464bdb734a19a7cf0c8569eb2c95&flowBeginTime=1574847318905&flowId=3fefe938bb86ff4c315e44aef31bbbf408adeb3f1b9111bbb2bc2cab7824d1de